presto-hive/src/test/java/com/facebook/presto/hive/metastore/thrift/TestThriftHiveMetastoreConfig.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.metastore.thrift;
+
+import io.airlift.units.Duration;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static io.airlift.configuration.testing.ConfigAssertions.assertRecordedDefaults;
+import static io.airlift.configuration.testing.ConfigAssertions.recordDefaults;
+import static org.testng.Assert.assertEquals;
+
+public class TestThriftHiveMetastoreConfig
+{
+    @Test
+    public void testDefaults()
+    {
+        assertRecordedDefaults(recordDefaults(ThriftHiveMetastoreConfig.class)
+                .setBackoffScaleFactor(2.0)
+                .setMaxBackoffDelay(new Duration(1, TimeUnit.SECONDS))
+                .setMaxRetries(10)
+                .setMaxRetryTime(new Duration(30, TimeUnit.SECONDS))
+                .setMinBackoffDelay(new Duration(1, TimeUnit.SECONDS)));
+    }
+
+    @Test
+    public void testExplicitParameterSetup()
```

Comment:
Why is this different from `com.facebook.presto.sql.analyzer.TestFeaturesConfig#testExplicitPropertyMappings`? I think this is supposed to be testing that Properties-provided (textual) configuration can be applied (converted).

@VicoWu did you get an explicit advice to implement the test this way?

Reply:
@findepi  
Sorry, it's my fault that I didn't obey the code convention in Presto;
I have re-submitted the code; Please help to review it;

Reply:
@findepi 
Sure. From my understanding, there will be a new version presto releasing in the coming days, I will have to wait until it released before this patch could be merged
I will ping you after that;